gradle.properties

```diff
@@ -12,7 +12,7 @@ kotlin.version.snapshot=1.3-SNAPSHOT
 # Also set KONAN_LOCAL_DIST enviroment variable in bootstrap mode to auto-assign konan.home
 
 junit_version=4.12
-infra_version=0.1.0-dev-42
+infra_version=0.1.0-dev-45
```

Comment:
There's a new `-dev-47` version which supports 1.3.30, 1.3.40 and master compilers